templates/includes/base/header.html

```diff
@@ -6,15 +6,18 @@
           developer
         </a>
     </div>
-
-    <nav id="navigation" class="p-navigation__nav u-float--right">
+    {% if 'core' in request.path %}{% else %}
```

Comment:
Can you create an issue to un-hardcode the core section here after this is merged, please. :)